x/ecocredit/server/core/create_class.go

```diff
@@ -25,13 +25,14 @@ func (k Keeper) CreateClass(goCtx context.Context, req *core.MsgCreateClass) (*c
 	}
 
 	// TODO: remove params https://github.com/regen-network/regen-ledger/issues/729
```

Comment:
Should this still be here or moved down with `k.paramsKeeper.Get` ?

Reply:
yeah should still be there, moved it down to the actual `Get` call